packages/session-replay-browser/src/hooks/beacon.ts

```diff
@@ -0,0 +1,67 @@
+import { getGlobalScope } from '@amplitude/analytics-client-common';
```

Comment:
Nit: this itself isn't a hook - could it go in a helpers folder within the hooks folder?

Reply:
Yeah that sounds good!